Formula/rabbitmq.rb

```diff
@@ -67,6 +68,7 @@ def rabbitmq_env
       NODE_IP_ADDRESS=127.0.0.1
       NODENAME=rabbit@localhost
       RABBITMQ_LOG_BASE=#{var}/log/rabbitmq
+      PLUGINS_DIR="#{prefix}/plugins:#{HOMEBREW_PREFIX}/share/rabbitmq/plugins"
```

Comment:
We understand `#{prefix}/plugins `to be the current default plugin directory.

Reply:
Where do they currently install?

Reply:
The RabbitMQ documentation says that they are in `/usr/local/Cellar/rabbitmq/{version}/plugins`
  https://www.rabbitmq.com/installing-plugins.html

The formula only writes this file if it doesn't already exist, so to bypass the version number changes we had to use the linked location `/opt/homebrew/opt/rabbitmq/plugins`

We verified that the plugins directories have been updated by running:

`rabbitmqctl eval 'application:get_env(rabbit, plugins_dir).'`

And we can see that the default plugins are still available when running:

`rabbitmq-plugins list`